src/safari/global.js

```diff
@@ -288,17 +288,17 @@ Zotero.Connector_Browser = new function() {
 		let isOnline = await Zotero.Connector.checkIsOnline();
 		let image, tooltip;
 		if (isOnline) {
-			image = "images/toolbar/zotero-new-z-16px.png";
+			image = "images/toolbar/zotero-z-32px.png";
 			tooltip = "Zotero is Online";
 		} else {
-			image = "images/toolbar/zotero-z-16px-offline.png";
+			image = "images/toolbar/zotero-z-32px-offline.png";
 			tooltip = "Zotero is Offline";
 		}
 		return [image, tooltip]
 	}
 
 	function _showTranslatorIcon(translator) {
-		let image = "images/toolbar/treesource-collection.png";
+		let image = "images/collection.png";
 		if (translator.itemType !== "multiple") {
 			image = Zotero.ItemTypes.getImageSrc(translator.itemType).replace('images/', 'images/toolbar/')
```

Comment:
`getImageSrc` already provides `images/toolbar` in the path, so if you replace it again here, the path becomes `images/toolbar/toolbar/item.png`. You could copy what I did in my PR to avoid making too many assumptions:
````js
let image = Zotero.ItemTypes.getImageSrc(translator.itemType === "multiple" ? "collection" : translator.itemType)
			.replace(`${safari.extension.baseURI}safari/`, '');
````
`getImageSource` already returns the correct path for a collection.

`.replace(`${safari.extension.baseURI}safari/`, '');` should also be deleted below.